CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * Removed `mnemonic.to_public_key` in favor of `account.address_from_private_key`.
 * Removed `algosdk.future` package.  Moved package contents to `algosdk`.
 * Removed `encoding.future_msgpack_decode` method in favor of `encoding.msgpack_decode` method.
+* Mark `LogicSig` class as hidden in favor of internal us, renaming it to `_LogicSig`.
```

Comment:
@ahangsu I _think_ there's a minor typo here.  I left a _suggestion_ with what I believe to be the intended phrasing.

```suggestion
* Mark `LogicSig` class as hidden in favor of internal use, renaming it to `_LogicSig`.
```